src/commands/rtc/apps/video/deploy.js

```diff
@@ -15,23 +15,18 @@ class DeployCommand extends TwilioClientCommand {
       }
     }
 
-    const appInfo = await getAppInfo.call(this);
+    this.appInfo = await getAppInfo.call(this);
 
-    if (appInfo) {
-      if (this.flags.override) {
-        await this.twilioClient.serverless.services(appInfo.sid).remove();
-        console.log(`Removed app with Passcode: ${appInfo.passcode}`);
-      } else {
-        console.log('A Video app is already deployed. Use the --override flag to override the existing deployment.');
-        await displayAppInfo.call(this);
-        return;
-      }
+    if (this.appInfo && !this.flags.override) {
+      console.log('A Video app is already deployed. Use the --override flag to override the existing deployment.');
+      await displayAppInfo.call(this);
+      return;
     }
     await deploy.call(this);
     await displayAppInfo.call(this);
   }
 }
-
+DeployCommand.appInfo = null;
```

Comment:
This looks odd.  Do we have to set `appInfo` as an instance variable, or can we just pass it as an argument to `deploy()`, just like how it's done with `getAssets(folder, appInfo)`.

Reply:
Agreed 👍

Reply:
Actually looking at this closer, this appears to be the convention for passing information. Note that the flags are passed as an instance variable to the deploy invocation. I'd prefer to keep the convention the same and change in a separate PR.

Reply:
Either way, I don't think this line is needed 😄

Reply:
Nit:

I don't think it's convention to create a new instance variable outside of a constructor.

But more importantly, passing `appInfo` as a function argument better expresses the intent of your code. 

When you set `this.appInfo = await getAppInfo.call(this);`, it doesn't say anything about where that value will be consumed.  There's nothing in `deploy.js` that tells you that the `deploy` function depends on `this.appInfo`. 

But when you pass it as an argument (`deploy.call(this, appInfo)`), it becomes clear that `appInfo` will be consumed by the `deploy` function. It's more explicit, and IMO more readable.

Reply:
I definitely agree with you, but I'd rather preserve the existing convention with these changes as opposed to changing it for this addition. The convention can be updated in a separate PR. The `helpers.deploy()` function currently accesses the following instance variables that are not passed to the function.

```
this.flags
this.twilioClient
this.appInfo // I added this one.
```

We can update the `deploy` method to take these parameters as arguments in a separate PR.